lib/src/dbs/options.rs

```diff
@@ -415,13 +415,11 @@ impl Options {
 	/// Get currently selected NS
 	pub fn ns(&self) -> &str {
 		self.ns.as_ref().map(AsRef::as_ref).unwrap()
```

Comment:
Unwrapping on an option can be a pain to debug when it panics because it doesn't say whether the `None` is unexpected or just poorly handled. I know this is not part of this PR but since we are removing a comment using `Error::Unreachable` below, can we address this here as well so we don't forget it?

```suggestion
		match &self.ns {
		    Some(ns) => ns.as_ref(),
		    None => unreachable!("the namespace should already be set at this point"),
		}
```

Reply:
Can't we just do `.expect(..)`?

Reply:
We can but `unreachable` is more semantically correct here because it explicitly says this branch shouldn't be reachable. You can quickly understand it even at a glance without even reading the message.